.github/workflows/main.yaml

```diff
@@ -257,9 +267,17 @@ jobs:
         uses: ./.github/actions/gn
       - name: Build Cobalt
         uses: ./.github/actions/build
+      - name: 'Check Artifact Size'
+        uses: ./.github/actions/check_artifact_size
+        if: ${{ inputs.check_artifact_size && matrix.sb_api_version == ''}} # Only check current SB version for Evergreen.
+        with:
+          workflow: ${{ inputs.check_artifact_size }}
+          name: ${{ matrix.platform }}-${{ matrix.config }}
+          path: out/${{ matrix.platform }}_${{ matrix.config }}/${{ inputs.keep_artifacts }}
```

Comment:
Technically we cannot directly replace the path with `out/${{ matrix.platform }}_${{ matrix.config }}/install/lib/${{ inputs.keep_artifacts }}` here, because there are other platforms relying on the `Upload Artifacts` action, such as android, and the Android APKs are not located in install directory. 
The point of adding `Check Artifact Size` is to compare the delta of size between PRs, Therefore, I think it might not matter much if the artifact is not stripped.

Reply:
I think i agree with Holden here, we do want to check stripped binaries only. If i reconfigure the builds to emit larger debugsymbols or the updated toolchain does, i don't think we want to be alerted here at all.

Stripped output on `gold` ( and maybe on `qa` ) builds is really the key metric to track.